src/redux/store.js

```diff
@@ -0,0 +1,23 @@
+import { combineReducers, createStore } from 'redux'
+import { favoriteDrinksReducer } from './reducers/favoriteDrinkReducer'
+import { addFavoriteDrink, removeFavoriteDrink } from './actions/favoriteDrinkAction'
+import { liquorSearchReducer } from './reducers/liquorSearchReducer'
+import { boardGameReducer } from './reducers/boardGameReducer'
+// import { loadDrinkData } from './actions/liquorSearch'
+
+const rootReducers = {
+    favoriteDrinks: favoriteDrinksReducer, 
+    liquorSearch: liquorSearchReducer,
+    boardGame: boardGameReducer
+}
+
+const store = createStore(combineReducers(rootReducers))
+
+export default store
+    store.subscribe(() => {
```

Comment:
While I haven't used Redux in a long time, I do believe that you only need to subscribe if you intend to _do something_ when a certain change is made. 

If you're using this so that you can easily see what's going on in Redux and when various actions are dispatched, I would encourage utilizing the Redux Devtools instead to limit extra code.

Reply:
Good call, we will definitely remove it.